util/exercise/src/cmd/configure.rs

```diff
@@ -21,8 +22,8 @@ fn get_user_input(prompt: &str) -> Result<String> {
 
 fn get_user_config(exercise_name: &str, config_content: &Value) -> Result<Value> {
     let existing_config = get!(config_content, "exercises", as_array)
-        .iter()
-        .find(|exercise| exercise["slug"] == exercise_name);
+        .par_iter()
```

Comment:
This doesn't feel like a case where parallelization is actually helpful: we have ~100 exercises in this list, each of which will be processed near-instantaneously.

Benchmarking this would tell us for sure, but my intuition is that Rayon won't actually use more than a single thread in this case, meaning that we pay the parallelization initialization overhead for no reason, meaning that this change actually slows things down.